docs/design.md

```diff
@@ -55,25 +55,51 @@ description: "My New APB"
 bindable: false
 async: optional
 parameters: []
+dependencies: []
 ```
 The spec file will need to be edited for your specific application.
 
 For example, the `etherpad-apb` spec file looks as follows:
 ```yml
-name: fusor/etherpad-apb
+name: etherpad-apb
+image: ansibleplaybookbundle/etherpad-apb
 description: Note taking web application
 bindable: true
 async: optional
+metadata:
+  displayName: "Etherpad (APB)"
+  longDescription: "An apb that deploys Etherpad Lite"
+  imageUrl: "https://translatewiki.net/images/thumb/6/6f/Etherpad_lite.svg/200px-Etherpad_lite.svg.png"
+  documentationUrl: "https://github.com/ether/etherpad-lite/wiki"
 parameters:
-  - name: hostport
-    description: The host TCP port as the external endpoint
-    type: int
-    default: 9001
-  - name: db_user
-    description: Database User 
-    type: string
+  - mariadb_name:
```

Comment:
Not necessarily relevant for this PR, but the way we do bind now would not use these variables.